_sass/components/homepage/_infocols.scss

```diff
@@ -0,0 +1,70 @@
+.infocols-container {
+  max-width: 1280px;
+  margin: auto;
+}
+
+.infocols-section {
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: 3rem;
+  flex-wrap: wrap;
+
+  .infocols-footer {
+    flex-direction: row;
+    gap: 0.75rem;
+
+    .infocols-linktext {
+      font-size: 1.125rem;
+      text-transform: uppercase;
+      margin: 0;
+      transform: translateY(
+        -0.09rem
+      ); // making it centered with the arrow-right icon
+    }
+  }
+}
+
+.infocols-infobox-section {
+  align-items: flex-start;
+  justify-content: center;
+  text-align: left;
+  gap: 2rem;
+  width: 100%;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  // Make it into row for tablet onwards
```

Comment:
Unfortunately we can't use the breakpoints definition directly from `_breakpoints.scss` but is it possible to put in the comment exactly which size this should be (i.e. `md` for this case)? Same for the other comments below.

This is so that it is clearer when and why we are breaking, and if for some reason we are adjusting those breakpoint values it is easier to find and understand.

Reply:
sure, added in the comments